templates/webpack.javascript.js

```diff
@@ -32,8 +32,8 @@ module.exports = env => {
         // The 'appPath' and 'appResourcesPath' values are fetched from
         // the nsconfig.json configuration file
         // when bundling with `tns run android|ios --bundle`.
-        appPath = "app",
-        appResourcesPath = "app/App_Resources",
+        appPath = "src",
```

Comment:
We changed the default paths only for the angular templates so far, ts/js/vue still use the app & app/App_Resources logic (e.g. https://github.com/NativeScript/template-blank-ts). We should update them first prior to updating the default webpack templates?

Reply:
I updated the PR to modify only the NG webpack config. After we migrate the other templates, I will open a PR that changes the non-NG webpack configs as well.